### PR TITLE
Feat home page

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
Home Page Added for Quiz App. Here I have added Navigation, main menu, and footer for the Quiz app.

### CODE REVIEW

**Review by kotalwar-krishna — APPROVED** [review]
Looks Good

File: style.css [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+}
+
+.quiz-type-one,
+.quiz-type-two,
+.quiz-type-three {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+
```
Comment (mariyasada) [inline style.css]: why are you repeating the same properties for question-type-one and question-type-two? combine both classes that have the same property.

File: style.css [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+    width: 20rem;
+    height: 15rem;
+    background-image: url(/assets/logical.jpg);
+    background-repeat: no-repeat;
+    background-attachment: scroll;
+    background-position: center center;
+    background-size: cover;
+    border-radius: 1rem;
+    color: var(--theme);
+}
```
Comment (mariyasada) [inline style.css]: here also

**Review by mariyasada — APPROVED** [review]
look at my comment

File: style.css (hunk 1 above)
Comment (Kothalkarpriya) [reply style.css]: Sure I will Do it

File: style.css (hunk 2 above)
Comment (Kothalkarpriya) [reply style.css]: I will do it

**Pull request merged**